WooCommerce/Classes/POS/Presentation/Reusable Views/POSPageHeaderView.swift

```diff
@@ -211,11 +210,12 @@ private enum Constants {
             .foregroundColor(.posOnSurface)
         }
 
+        @State var isProductsSelected: Bool = true
```

Comment:
nit: We get a warning for using `@State` within the Preview macro, it seems we have to move this to the root of the Preview and tag it as well with `@Previewable`